src/store/audio.ts

```diff
@@ -21,6 +21,20 @@ async function generateUniqueId(audioItem: AudioItem) {
     .join("");
 }
 
+function buildFileName(state: State, audioKey: string) {
+  // eslint-disable-next-line no-control-regex
+  const sanitizer = /[\x00-\x1f\x22\x2a\x2f\x3a\x3c\x3e\x3f\x7c\x7f]/g;
```

Comment:
0x5C（\）でエラーが出たので追加していただけると。
```suggestion
  const sanitizer = /[\x00-\x1f\x22\x2a\x2f\x3a\x3c\x3e\x3f\x5c\x7c\x7f]/g;
```